### DISCUSSION TITLE
Discourse Proposal

### DISCUSSION THREAD

**Discussion by mgrover1** [openradar/erad2022#68] [Ideas]
As was mentioned during the meeting, we would like to setup a discourse for the open radar community. @kmuehlbauer mentioned that they have a free resource application for open source projects ([see the application page here](https://blog.discourse.org/2018/11/free-hosting-for-open-source-v2/)).

We should draft up a proposal for the open radar science organization.

**Comment by kmuehlbauer** [comment]
I took the chance and applied already.They will come back to me within two days .

**Comment by mgrover1** [comment]
Woo! Great!

**Comment by kmuehlbauer** [comment]
Unfortunately I've got no response from the discourse people until now beside the initial confirmation email (2022 August 30). In that response they promised "in any case, you'll hear from us soon". 

I've sent a follow up email on September 1, explaining the new layout of openradarscience.org, which changed some url paths. Not sure how to proceed now. Restart from scratch?

**Reply by mgrover1** [reply]
Another option could be to chat with the Pangeo community and see if we can align with their efforts. They mentioned they are open to non-Python efforts. They already have a large community there, focused around open science, open infrastructure, and open communication.

**Reply by kmuehlbauer** [reply]
I've sent another follow-up mail to the discourse team, adding their main email contact and asked for a reply. Nevertheless it would be good to have a second option with the pangeo folks.
